.github/workflows/runtests.yml

```diff
@@ -90,44 +88,41 @@ jobs:
   # -------------------- Test split 1/2 latest Python -------------- #
 
   tests1-latest:
-    needs: code-quality
 
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
-          python-version: '3.9'
+          python-version: '3.11.0-alpha.2'
```

Comment:
The setup-python README suggests that you can use a range here like
```
python-version: '3.11.0-alpha - 3.11.0'
```
I presume that means that it will use the newest version until 3.11.0. Maybe it's possible to change it to 3.11 so it will continue using newest versions of 3.11 after 3.11.0 is released...

https://github.com/actions/setup-python

Reply:
OK! I'll try it! (I noticed that alpha-4 was available so I was about to change anyway.)

Reply:
Seems to work!

(Matplotlib seems to have a harder time with alpha-4 compared to alpha-2 though... Or maybe the actual install order is randomized in some way...)

Reply:
Or rather: there is a cached numpy wheel for alpha-2. But should just be a question of time before there is one for alpha-4 as well and we are back to scipy and kiwisolver causing issues (which seems to be the ones having problems on the latest alpha-2 build).